lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -658,6 +716,7 @@ WasmBinaryReader::LEB128(UINT &length, bool sgn)
     UINT result = 0;
     UINT shamt = 0;
     byte b;
+    length = 0;
```

Comment:
@commonlisp  should this be initialized to 1? It seems small number returns 0 for length.